3.11/http/aql-query-cursor-query-results.md

```diff
@@ -171,6 +171,56 @@ Content-type: application/json
 }
 ```
 
+If the attribute `retriable` is set to true, the response object will contain the attribute `nextBatchId`,
+unless the last batch has already been fetched (meaning `hasMore` is false). 
+This attribute can be used if, upon fetching the next batch before advancing the cursor again, if this 
```

Comment:
I have a hard time understanding this sentence. Can we shorten it to make it more understandable?